docs/references/nextjs/overview.mdx

```diff
@@ -1,13 +1,13 @@
 ---
 title: Clerk Next.js SDK
-description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your Next.js application.
+description: Learn how to integrate Clerk into your Next.js application using the Clerk Next.js SDK.
 ---
 
-Clerk makes it simple to add authentication to your Next.js application. This reference lists the features and methods available in Clerk's Next.js SDK.
+The Clerk Next.js SDK is the recommended method for integrating Clerk into your Next.js application.
 
 ## Client-side helpers
 
-Because the Next.js SDK is built on top of the Clerk React SDK, you can utilize the hooks that the React SDK provides. These hooks give you access to the [`Clerk`](/docs/references/javascript/clerk/clerk) object and a set of useful helper methods for signing in and signing up. Learn more about these hooks in the [React SDK reference](/docs/references/react/overview).
```

Comment:
I think this description is helpful because when the users click on the links to the hooks, they're going to be confused why they just got taken to the React SDK docs. For now, we need to keep this context

Reply:
https://github.com/clerk/clerk-docs/pull/1815/commits/a8a1b5b1b89e127b992bb55417394db4181c6afc